upgrade-tests/src/test/java/io/zeebe/test/RollingUpdateTest.java

```diff
@@ -52,6 +58,19 @@
           .endEvent()
           .done();
 
+  private static final File SHARED_DATA;
+
+  static {
+    final var sharedDataPath =
+        Optional.ofNullable(System.getenv("ZEEBE_CI_SHARED_DATA"))
```

Comment:
Why do you need this "ZEEBE_CI_SHARED_DATA"?

Reply:
We've had issues with sharing data folders with the Docker sibling container before in CI. If you mounted it to `/tmp` it would fail to mount with read/write access so the test container and the docker container wouldn't be able to see the same files, which is important if you want to check for snapshots, for example.

So, when testing locally, you do want to use `/tmp` since it's the standard tmp folder where things are cleaned up after. But on CI, you want to use another well defined mount points (which may not exist on the local dev machine, or may not get automatically cleaned up after), so that's what we use. The env var is then set in the Jenkinsfile.

Reply:
:+1:

Reply:
Might worth to add a comment